src/providers/vod/vod-provider.ts

```diff
@@ -61,9 +59,10 @@ export class VodProvider extends Provider {
           const thumbCuePointsLoader: ThumbLoader = data.get(ThumbLoader.id);
           const thumbCuePoints : Array<KalturaThumbCuePoint> = thumbCuePointsLoader?.response.thumbCuePoints || [];
           this._logger.debug(`_fetchVodData response successful with ${thumbCuePoints.length} cue points`);
-
+          const ks = this._player.config.session.ks || "";
+          const serviceUrl = this._player.config.provider.env.serviceUrl || "";
```

Comment:
env is nullable

Reply:
so if service url is empty we should fail and not call the network cause it will create unexpected urls.
Same for KS, if it is mandatory then lets also add to validation, so we check it with thumbCuePoints length